python/ray/worker.py

```diff
@@ -1730,7 +1735,10 @@ def init(redis_address=None,
          plasma_directory=None,
          huge_pages=False,
          include_webui=True,
-         use_raylet=None):
+         use_raylet=None,
+         configure_logging=True,
+         logging_level=logging.INFO,
+         logging_format="%(message)s"):
```

Comment:
This string appears in a lot of places. Maybe we should move it to `ray_constants.py`.